src/main/java/com/google/api/generator/gapic/model/Method.java

```diff
@@ -19,16 +19,38 @@
 
 @AutoValue
 public abstract class Method {
+  public enum Stream {
+    NONE,
+    CLIENT,
+    SERVER,
+    BIDI
```

Comment:
Can I have some context, what is `BIDI` :)

Reply:
**BiDi**rectional streaming, I think.